### ISSUE TITLE
Add Object Detection Project !! 

### ISSUE THREAD

**Issue by BlueBlaze6335** [gdscjgec/ML-Gallery#10]
### You can add any model , either train it or use pretrained weights for generating best results !! Create proper documentation supporting your choice of model , and put that in a README !! Your project should be contained in a directory - "Object Detection" 
and if you have any files or images along with your project then put that in a folder - ```Object Detection/Assets``` !! Happy contributing !!

**Comment by IshitaSaha3002** [comment]
Hey! Can I go ahead with this?

**Comment by BlueBlaze6335** [comment]
@IshitaSaha3002 sure !! Just go through the repo and see how the README and other documentation is done !! Then submit your PR when you are done. Happy contributing !!

**Comment by varun290101** [comment]
Can I try this?

**Comment by BlueBlaze6335** [comment]
@varun290101 yes you can !!!

**Comment by varun290101** [comment]
@BlueBlaze6335 Can you recommend a dataset for this please?

**Comment by BlueBlaze6335** [comment]
@varun290101 there are many - CIFAR10 OR CIFAR1000 , Imagenet , darknet , coco.

**Comment by varun290101** [comment]
Bhaiya, I just saw that there was already ann vs cnn code on CIFAR10, I did it on the same thing. It cant be pulled na

**Comment by BlueBlaze6335** [comment]
No , there is no problem in doing that. You can use it !!

**Comment by Kasrazeba-Mulla** [comment]
@BlueBlaze6335 should i add the iris dataset for object recognition in Datasets.md ? Currently , it has no dataset links for object detection

**Comment by Kasrazeba-Mulla** [comment]
@BlueBlaze6335 can i work on fruit detection dataset ? Also , do i need to add the dataset link to datasets.md ?

**Comment by puranjay123** [comment]
@BlueBlaze6335 can you assign me this.

**Comment by BlueBlaze6335** [comment]
@puranjay123 sure !!

**Comment by BlueBlaze6335** [comment]
@Kasrazeba-Mulla check [this](https://www.analyticsvidhya.com/blog/2021/05/top-10-open-source-datasets-for-object-detection-machine-learning-in-2021/) out.

Please dont forget to add the dataset link in your readme and also update the Dataset.md if the link is not present !!

**Comment by Kasrazeba-Mulla** [comment]
@BlueBlaze6335 thank you 
Yes i will add the dataset and double check this time before making a pr

**Comment by BlueBlaze6335** [comment]
@Kasrazeba-Mulla 👍

**Comment by BlueBlaze6335** [comment]
Everyone please check the discussion section !!

**Comment by BlueBlaze6335** [comment]
@Yasas4D go ahead !!

**Comment by puranjay123** [comment]
@BlueBlaze6335 you have marked as spam my PR's and now I am banned from hacktoberfest can you undo that.

**Comment by BlueBlaze6335** [comment]
@puranjay123 you have spammed PRs so I have labelled them as invalid. I have removed one. And I guess you have spammed PRs in other projects too. That has lead to your ban !! Even after doing good projects , if you don't follow the rules , you will face such situations. Make sure you don't do this next time !!

**Comment by puranjay123** [comment]
@BlueBlaze6335  No, i do not have any other PR that was marked as spam just the PR that you marked as spam. Now I am banned from contributing to hacktoberfest for lifetime. :) I was just new to opensource and this is the worst experience I have ever had.
Please next time carefully mark someone PR spam it could lead to ban for the lifetime.

**Comment by BlueBlaze6335** [comment]
@puranjay123 if your pr was a spam , I will mark it as a spam ! And just one or two invalid labels doens't get you banned. If you are new , you should follow the rules. You did not follow them ,you are facing this. You do your work as a contributor properly , and you better fix this attitude.

**Comment by puranjay123** [comment]
@BlueBlaze6335 sorry if you feel it that way I did not have the intent of that way of hurting you. , I am just saying that you can just reject it.
Sorry if you felt it that way . Next time I will mark my mistakes and will work on them as a contributor.
Sorry again.

**Comment by BlueBlaze6335** [comment]
See , there are a few rules that I need to abide by. If the pr is not valid , I have to mark it invalid. I gave warnings. So if you are doing something for the first time make sure you are following the rules.

**Comment by puranjay123** [comment]
got it .
